dap_aria_mapping/getters/validation.py

```diff
@@ -0,0 +1,37 @@
+from nesta_ds_utils.loading_saving.S3 import download_obj
+from dap_aria_mapping import BUCKET_NAME
+
+def get_tree_depths() -> dict:
+    """gets a dictionary with metrics about the tree depth of the taxonomies
+
+    Returns:
+        dict: size and depth metrics of trees in taxonomy
+    """
+    return download_obj(
+        BUCKET_NAME,
+        'outputs/validation_metrics/taxonomy_depth.json',
+        download_as = "dict")
+
+def get_entropy() -> dict:
+    """gets a dictionary with metrics about the entropy of the distribution of 
+        entites/category at a given level of the taxonomy
+
+    Returns:
+        dict: key: taxonomy name, value: dict: key: level, value: entropy of frequency distribution
```

Comment:
I would encourage hinting with typing's Dict, and in the docstring something like
`{taxonomy_name: {level: entropy_val,
        ...
    },
    ...
}`